mne/preprocessing/ica.py

```diff
@@ -2624,3 +2638,39 @@ def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
         return template_fig, labelled_ics
     else:
         return None
+
+
+def read_eeglab_ica(file_name):
```

Comment:
Since we already have `read_ica` I suggest to rename this function to `read_ica_eeglab` (in line with `read_raw_*` conventions).

Reply:
Sure, make sense.